state-chain/pallets/cf-broadcast/src/lib.rs

```diff
@@ -832,6 +835,13 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 				broadcast_id,
 			);
 
+			T::CfeEventEmitter::tx_broadcast_request(TxBroadcastRequest {
+				broadcast_id,
+				nominee: nominated_signer.clone(),
+				payload: broadcast_attempt.transaction_payload.clone(),
+			});
+
+			// TODO: consider removing this
```

Comment:
Why would we not remove it? I guess polkaJS, debugging?

Reply:
*edit - I saw there was another comment thread about this, think I agree with keeping it for now*